views/ok.ejs

```diff
@@ -1,35 +0,0 @@
-<html>
-<head>
-    <meta charset="utf-8">
-    <link href="bower_components/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
-    <script type="text/javascript" src="bower_components/jquery/dist/jquery.min.js" charset="utf-8"></script>
-
-</head>
-<body>
-<script type="text/javascript" src="http://localhost:3000/bundle.js" charset="utf-8">
-</script>
-<script>
-    var ludwig = new Ludwig();
-</script>
-
-</a>
-</body>
-
-<body>
-
-<!-- Begin page content -->
-<div class="container">
-    <h1>Merci!</h1>
-    <p>Votre suggestion a bien été enregistrée, elle est accessible en suivant <a href="<%=pullRequestURL%>">ce lien</a></p>
```

Comment:
>  Because it is a way to mimic an API workflow.

Yes… but is it exactly what we want here? As seen yesterday IRL: we want end users to suggest improvements. The more Ludwig users have to set up in their app, the less they will use it. I think Ludwig should provide a default OK view. Yes, it should definitely be overridable, but removing code that was already there seems like a bad move.